mne/decoding/tests/test_transformer.py

```diff
@@ -0,0 +1,156 @@
+# Author: Mainak Jas <mainak@neuro.hut.fi>
+#         Romain Trachel <trachelr@gmail.com>
+#
+# License: BSD (3-clause)
+
+import warnings
+import os.path as op
+import numpy as np
+
+from nose.tools import assert_true, assert_raises
+from numpy.testing import assert_array_equal
+
+from mne import io, read_events, Epochs, pick_types
+from mne.decoding import Scaler, FilterEstimator
+from mne.decoding import PSDEstimator, ConcatenateChannels
+
+warnings.simplefilter('always')  # enable b/c these tests throw warnings
+
+tmin, tmax = -0.2, 0.5
+event_id = dict(aud_l=1, vis_l=3)
+start, stop = 0, 8
+
+data_dir = op.join(op.dirname(__file__), '..', '..', 'io', 'tests', 'data')
+raw_fname = op.join(data_dir, 'test_raw.fif')
+event_name = op.join(data_dir, 'test-eve.fif')
+
+
+def test_scaler():
+    """Test methods of Scaler
+    """
+    raw = io.Raw(raw_fname, preload=False)
+    events = read_events(event_name)
+    picks = pick_types(raw.info, meg=True, stim=False, ecg=False,
+                       eog=False, exclude='bads')
+    picks = picks[1:13:3]
+
+    epochs = Epochs(raw, events, event_id, tmin, tmax, picks=picks,
+                    baseline=(None, 0), preload=True)
+    epochs_data = epochs.get_data()
+    scaler = Scaler(epochs.info)
+    y = epochs.events[:, -1]
+
+    # np invalid divide value warnings
+    with warnings.catch_warnings(record=True):
```

Comment:
insert :

warnings.simplefilter('always')  # enable b/c these tests throw warnings

here. Not at top of the file.